nengo_gui/static/components/netgraph_item.js

```diff
@@ -356,8 +356,11 @@ Nengo.NetGraphItem.prototype.generate_menu = function () {
         }
     }
     if (this.sp_targets.length > 0) {
-        items.push(['Semantic pointer',
+        // Naming things is hard
+        items.push(['Semantic pointer similarity text',
                     function() {self.create_graph('Pointer', self.sp_targets[0]);}])
+        items.push(['Semantic pointer similarity graph',
```

Comment:
Maybe it is helpful to somehow add the word time (or a synonym) into the name to make clear that it shows the similarity over time (whereas the other plot only shows one specific time point).